bigquery/google/cloud/bigquery/job.py

```diff
@@ -1277,6 +1297,170 @@ def from_api_repr(cls, resource, client):
         job._set_properties(resource)
         return job
 
+    @property
+    def query_plan(self):
```

Comment:
Do you think we should create a separate QueryStatistics class to hold these read-only properties to match the REST API more closely?